app/src/main/java/com/io/movio/domain/GetMovieByIdUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.io.movio.domain
+
+import com.io.movio.data.models.Movie
+import com.io.movio.data.repositories.MovieRepository
+
+class GetMovieByIdUseCase(private val movieRepository:MovieRepository){
+
+    operator fun invoke(id: Int):Movie{
+        return getMovieById(id)
+    }
+
+    fun getMovieById(id:Int):Movie{
+      return movieRepository.getMovies()[id]
```

Comment:
this should be a different function, searching through the list for the given ID